.github/workflows/ci.yml

```diff
@@ -31,5 +52,15 @@ jobs:
       with:
         github-access-token: ${{ secrets.NIX_GIT_TOKEN }}
 
+    - name: Setup Caches
+      if: ${{ needs.check-user.outputs.isFloxUser == 'true' }}
+      uses: cachix/cachix-action@v12
+      with:
+        authToken: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        signingKey: ${{ secrets.FLOX_STORE_PUBLIC_NIX_SECRET_KEY }}
+        cachixArgs: --hostname s3://flox-store-public
```

Comment:
does not seem to work with this cachix version

Reply:
Yeah I'd say we should either use cachix or copy their `ls /nix/store` thing to use with `nix store copy` but I've got other stuff to work on.